src/components/entity/state-info.js

```diff
@@ -79,6 +100,15 @@ class StateInfo extends PolymerElement {
               datetime="[[stateObj.last_changed]]"
             ></ha-relative-time>
           </div>
+          <div class="last-updated">
+            <div class="key">Last updated</div>
+            <div class="value">
+              <ha-relative-time
+                hass="[[hass]]"
+                datetime="[[stateObj.last_updated]]"
+              ></ha-relative-time>
+            </div>
+          </div>
```

Comment:
```suggestion
            <paper-tooltip animation-delay="0">
              [[localize('ui.dialogs.more_info_control.last_updated')]]:
              <ha-relative-time
                hass="[[hass]]"
                datetime="[[stateObj.last_updated]]"
              ></ha-relative-time>
            </paper-tooltip>```

Reply:
I put it in span, so it won't be centered horizontally over screen, but appear below `ha-relative-time`